assets/app/util/documentStrategy.js

```diff
@@ -0,0 +1,50 @@
+import { decodeB64 } from './encoding';
+
+const frontmatterDelimiterRegex = /^---\n([\s\S]*?)---\n/;
+
+const initializeYml = content => {
+  return {
```

Comment:
There are a lot of `return {}` statements in this file. They all have different permutations so I'm not sure if there is a way to cut down their number